demisto_sdk/commands/common/tools.py

```diff
@@ -2006,13 +2020,13 @@ def get_content_path() -> Union[str, PathLike, None]:
 
         if not is_fork_repo and not is_external_repo:
             raise git.InvalidGitRepositoryError
-        return git_repo.working_dir
+        return Path(git_repo.working_dir).absolute()  # type: ignore[arg-type]
```

Comment:
why ignoring `arg-type`?

Reply:
`Path` expects `str` in `__init__`, however `git_repo.working_dir` could be `None`, `str` or `Path`.

Reply:
Can we handle the `None` case here?

Reply:
handled this case. Will return cwd in such case

Reply:
can we remove arg-type then? :)

Reply:
already removed